tests/trackingtest.py

```diff
@@ -0,0 +1,404 @@
+import absl.logging
+import argparse
+import os
+import json
+import logging
+import math
+import sys
+from pathlib import Path
+from ml_tools import tools
+from config.config import Config
+from classify.clipclassifier import ClipClassifier
+from .testconfig import TestConfig
+from .api import API
+
+MATCH_ERROR = 1
+
+
+def match_track(gen_track, expected_tracks):
+    score = None
+    match = None
+    MAX_ERROR = 8
+    for track in expected_tracks:
+        start_diff = abs(track.start - gen_track.start_s)
```

Comment:
This is clever - but is it really needed?   Can we just order the tracks by time and the compare the first with the first, etc.   If there is a different number of tracks chances are we will need to manually inspect the tracking to see if it ok.   

The key here is the normally we would expect the computed tracks to be very very similar to the saved tracks.   If they aren't we probably need to compare the two tracks to see which is right.

Reply:
You are probably right, i was doing it just by ordering before but in some cases were getting mis matches. But i am happy to say that these cases should be manually checked

Reply:
I think the case when this is important is if we are testing for false-positives. So if the test recording should have 1 valid track, but it is a noisy track.
When we adjust tracking parameters we may increase the noise and get a lot of false positive tracks. In this case comparing by tracking time is not going to work well.